account.go

```diff
@@ -262,20 +320,23 @@ func (p *PayoutScheduleParams) AppendTo(body *form.Values, keyParts []string) {
 
 // AccountParams are the parameters allowed during account creation/updates.
 type AccountParams struct {
-	Params                `form:"*"`
-	AccountToken          *string                       `form:"account_token"`
-	BusinessProfile       *AccountBusinessProfileParams `form:"business_profile"`
-	BusinessType          *string                       `form:"business_type"`
-	Company               *AccountCompanyParams         `form:"company"`
-	Country               *string                       `form:"country"`
-	DefaultCurrency       *string                       `form:"default_currency"`
-	Email                 *string                       `form:"email"`
-	ExternalAccount       *AccountExternalAccountParams `form:"external_account"`
-	Individual            *PersonParams                 `form:"individual"`
-	RequestedCapabilities []*string                     `form:"requested_capabilities"`
-	Settings              *AccountSettingsParams        `form:"settings"`
-	TOSAcceptance         *AccountTOSAcceptanceParams   `form:"tos_acceptance"`
-	Type                  *string                       `form:"type"`
+	Params          `form:"*"`
+	AccountToken    *string                       `form:"account_token"`
+	BusinessProfile *AccountBusinessProfileParams `form:"business_profile"`
+	BusinessType    *string                       `form:"business_type"`
+	Capabilities    AccountCapabilitiesParams     `form:"capabilities"`
```

Comment:
should this be a pointer? *AccountCapabilitiesParams

Reply:
it totally should, nice catch.

Reply:
fixed and tested locally